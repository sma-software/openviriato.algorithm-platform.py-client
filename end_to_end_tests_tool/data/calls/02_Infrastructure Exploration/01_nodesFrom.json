{
    "restResource": "/nodes",
    "httpMethod": "GET",
    "urlParameters": null,
    "queryParameters": [
        {
            "key": "HasSectionTrackFromNodeID",
            "value": "161",
            "optional": true
        }
    ],
    "requestBody": null,
    "CSharpMethodName": "GetNodesWithSectionTrackFrom(fromNodeID)",
    "CSharpMethodParameterTypes": [ "System.Int64" ],
    "subcategory": "Get Neighboring Nodes"
}
