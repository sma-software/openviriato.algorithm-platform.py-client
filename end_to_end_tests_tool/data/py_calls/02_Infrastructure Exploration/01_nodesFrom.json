{
    "PythonParameterMapping": {
      "from_node_id" : "$.queryParameters.[0].value"
    }
}
