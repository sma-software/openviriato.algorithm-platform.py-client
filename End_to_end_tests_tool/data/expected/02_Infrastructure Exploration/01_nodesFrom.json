[
    {
        "Code": "0085HED",
        "DebugString": "station:0085HED",
        "ID": 158,
        "NodeTracks": [
            {
                "ID": 159,
                "Code": "1",
                "DebugString": "stationtrack:0085HED_1"
            },
            {
                "ID": 160,
                "Code": "2",
                "DebugString": "stationtrack:0085HED_2"
            }
        ]
    },
    {
        "Code": "0085BONS",
        "DebugString": "station:0085BONS",
        "ID": 51,
        "NodeTracks": []
    }
]

